.gitignore

```
@@ -161,7 +161,7 @@ cython_debug/
 
 # this file is written automatically by setuptools_scm and should not be
 # tracked
-src/ccn_template/version.py
+src/ccn_template/ciao_version.py
```

```suggestion
src/ccn_template/version.py
```

I assume this was left over from testing.